src/pybind/mgr/progress/module.py

```diff
@@ -6,12 +6,11 @@
 import json
 
 
-# How often to potentially write back any dirty events (event history
-# persistence is best effort!
-PERSIST_PERIOD = 5
-
 ENCODING_VERSION = 1
 
+# keep a global reference to the module so we can use it from Event methods
+_module = None
```

Comment:
I didn't do the second one because EVent is the superclass of RemoteEvent that is used in other modules... which won't have  a ref for the progress module.

Note that I think the Event types should all get moved to mgr_module.py in a cleanup and progress.py should just have the logic for fabricating interesting progress events and doing the CLI.